### ISSUE TITLE
Make Agglomeration Plate Multiblock Ignore Direction

### ISSUE THREAD

**Issue by whizzball1** [quat1024/BotaniaTweaks#39]
it don't work

and it only matters because YAEH

Shadows: "if you accepted a proper state predicate direction could be ignored"

**Comment by quat1024** [comment]
how do i do state predicates with crafttweaker

**Comment by whizzball1** [comment]
It's not in CrT that you need to do it. It's in https://github.com/quat1024/BotaniaTweaks/blob/master/src/main/java/quaternary/botaniatweaks/modules/botania/recipe/AgglomerationRecipe.java
In multiblockmatches

**Comment by quat1024** [comment]
Ok, I'm not really sure how state *predicates* can help me here but I did play around with removing "direction"-like properties before comparing the iblockstates. Should work OK. I tried placing stairs in a bunch of different directions.

They do all get getdefaultstate-d if you use e.g. stairs as a multiblock *replacement* - I guess I could copy over state properties? not sure if people care enough for that though.
